_episodes_rmd/04-ggplot2.Rmd

```diff
@@ -62,20 +62,24 @@ this fashion allows for extensive flexibility and customization of plots.
 To build a ggplot, we will use the following basic template that can be used for different types of plots:
 
 ```
-ggplot(data = <DATA>, mapping = aes(<MAPPINGS>)) +  <GEOM_FUNCTION>()
+<DATA> %>%
+    ggplot(aes(<MAPPINGS>)) +
+    <GEOM_FUNCTION>()
 ```
+Remember from the last lesson that the pipe operator `%>%` places the result of the previous line(s) into the first argument of the function. **`ggplot`** is a function that expects a data frame to be the first argument. This allows for us to change from specifying the `data =` argument within the `ggplot` function and instead pipe the data into the function.
```

Comment:
This may be too verbose, but I think this clarifies your issue @juanfung . Thanks @atheobold for the help with the language here.